awscli/customizations/configure/list.py

```diff
@@ -105,15 +105,35 @@ def _lookup_credentials(self):
                 secret_key.mask_value()
                 return access_key, secret_key
 
-    def _lookup_config(self, name):
+    def _lookup_region(self):
+        region = self._lookup_in_env_and_config('region')
+        if region:
+            return region
+        region = self._session.get_config_variable('region')
+        if region:
+            # This makes the assumption that the only other possible place to
+            # retrieve region in the CLI is via IMDS. If we add more region
+            # providers in the future this will have to be updated
+            return ConfigValue(region, 'imds', '')
```

Comment:
Can we add a todo is try and come up with consistent location names?  It seems odd that we have `iam-role` for the creds but `imds` for the region even though they basically come from the same source.

Reply:
Sounds good. I'll add a todo before merging.